evm/chains/sharding/mainchain_handler/config.py

```diff
@@ -0,0 +1,9 @@
+# FIXME: should be 25 or 2500, fix it, only for faster testing
+SHUFFLING_CYCLE_LENGTH = 5
+DEPOSIT_SIZE = 10 ** 20
+PERIOD_LENGTH = 5
+
+TX_GAS = 510000
+GASPRICE = 1
+PASSPHRASE = '123'
+DEFAULT_RPC_SERVER_URL = 'http://localhost:8545'
```

Comment:
I'm thinking that this and `PASSPHRASE` don't belong as constants.  Currently it looks like this is primarily a testing configuration value.  If you modify the `MainChainHandler` class to instead take a fully instantiated `Web3` instance then we gain the flexibility to run tests how we wish as well as allow use of any style of web3 connection.

Reply:
Thank you for pointing this out. I moved `PASSPHRASE` to the tests and make `Web3` instance as the argument of the constructor of `MainchainHandler`.